examples/rust/components/blobby/src/lib.rs

```diff
@@ -1,117 +1,73 @@
-mod bindings {
-    use crate::Blobby;
+use ::std::num::ParseIntError;
```

Comment:
Nit: Do we need the fully qualified imports here? Makes sense for `http` because of the imports from the component library (which are also probably fine but could collide due to bindings)

Reply:
Nah not here, I just did it to be consistent with all the imports since I used `::http`, but I could just use non-qualified if you think that would be better

Reply:
It is really just a nit for now. Generally you only see that to disambiguate or in a macro. Your choice